.github/workflows/verify-dashboards-alerts.yaml

```diff
@@ -0,0 +1,135 @@
+name: Verify Dashboards and Alerts OK
+
+on: 
+  push:
+    branches: main
+    paths:
+       # Dashboards
+       - examples/dashboards/app_developer.json
+       - examples/dashboards/business_user.json
+       - examples/dashboards/platform_engineer.json
+       # Alerts
+       - examples/alerts/prometheusrules_policies_missing.yaml
+       - examples/alerts/slo-availability.yaml
+       - examples/alerts/slo-latency.yaml
+jobs:
+  verify-dashboards-alerts:
+    name: Verify Dashboards and Alerts OK
+    runs-on: ubuntu-latest
+    defaults: 
+      run:
+        shell: bash -eo pipefail {0}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run Quickstart
+        run: |
+          sudo chown $(whoami) .
+
+          sudo apt-get update -y
+          sudo apt-get install -y golang expect
+
+          export ISTIO_INSTALL_SAIL=true
+
+          cat << 'EOF' > automate_setup.expect
```

Comment:
Does those tests even require quickstart? You are only checking presence of both alerts and dashboads, you are never actually using Kuadrant

Reply:
@pehala I guess it is a simple way of being able to run grafana and prometheus. You are right that there are several components not in use.

Reply:
I have determined that it is possible to just set up the observability stack on its own, meaning only the required components would be ran. It would mean no dependency on the quickstart script and the runner-generated expect script. Implementing now...

Reply:
https://github.com/Kuadrant/kuadrant-operator/actions/runs/9838597648/job/27158825631